web/index.html

```diff
@@ -10,5 +10,11 @@
   <body>
     <div id="root"></div>
     <script type="module" src="/src/main.tsx"></script>
+    <script>
```

Comment:
Move this above loading main.tsx in order to ensure that no analytics events fire off before the analytics global is created